src/Directory.Build.props

```diff
@@ -20,6 +20,8 @@
 
     <ProduceReferenceAssembly Condition="'$(IsTestProject)' != 'true'">true</ProduceReferenceAssembly>
 
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```

Comment:
You shouldn't need this. Are you on RC2 locally?

Reply:
Just realized when you responded that you only need to be on RC1 for this, and dogfood VS should give it to you automagically.

Reply:
@BenVillalobos, have you forgotten to remove this line from PR? Please do it before I merge.

Reply:
Whoops, removed it. Thanks for double checking!